flax/experimental/nnx/nnx/graph.py

```diff
@@ -913,6 +913,30 @@ def update(
     )
     return unflatten(new_graphdef, state, idxmap=index_to_ref)[0]
 
+  def clear_refmap(self):
+    if self.refmap is None:
+      raise ValueError('Cannot clear refs without refmap.')
+
+    for ref in self.refmap:
+      if isinstance(ref, Variable):
+        ref.raw_value = None
```

Comment:
Does it make sense to add a `clear()` method to `Variable` doing this?

Reply:
I've removed this code, found a better way.